app/graphql/types/query_type.rb

```diff
@@ -354,4 +354,24 @@ def find_library_event_by_id(id:)
   def find_post_by_id(id:)
     RecordLoader.for(::Post, token: context[:token]).load(id)
   end
+
+  field :franchises, Types::Franchise.connection_type, null: true do
+    description 'All Franchise in the Kitsu database'
+  end
+
+  def franchises
+    ::Franchise.all
+  end
+
+  field :search_franchise_by_title, Types::Franchise.connection_type, null: true do
+    description <<~DESCRIPTION.squish
+      Search for Franchise by title using Algolia.
+      The most relevant results will be at the top.
+    DESCRIPTION
+    argument :title, String, required: true
+  end
+
+  def search_franchise_by_title(title:)
+    # NOTE: add Algolia search to allow for this.
```

Comment:
Needs to be completed

Reply:
@NuckChorris

Reply:
Let's skip this for now, I'll add it in a later PR